src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -2502,6 +2502,22 @@ bool QgsPostgresProvider::addFeatures( QgsFeatureList &flist, Flags flags )
         {
           v = paramValue( value.toString(), defaultValues[ i ] );
```

Comment:
Could this be moved to an `else` statement below to avoid immediately reassigning it in the case where `value` is a `StringList` or `List`?

Reply:
Yup, fair point, flows better that way I think